src/temperature_tele.rs

```diff
@@ -0,0 +1,47 @@
+//! # Thermostat_EEM temperature telemetry processing
+
+use serde::Serialize;
+
+/// Temperature telemetry struct.
+#[derive(Serialize, Copy, Clone, Default, Debug)]
+pub struct Temperature {
+    average: f32,
+    min: f32,
+    max: f32,
+}
+
+/// Temperature buffer for computing min/max/average of the last telemetry period.
+#[derive(Default, Copy, Clone, Debug)]
+pub struct TemperatureBuffer {
+    min: f32,
+    max: f32,
+    accumulator: f32,
+    counter: u32,
+}
```

Comment:
`stddev` is also very useful, but should probably be done in a separate PR, as you'll first need to look up the right algo.

Reply:
ack.

Reply:
Do you have a resource for this?

Reply:
Is [this](https://en.wikipedia.org/wiki/Algorithms_for_calculating_variance#:~:text=to%20a%20degree.-,Welford%27s%20online%20algorithm,-%5Bedit%5D) the one you have in mind?

Reply:
https://en.wikipedia.org/wiki/Algorithms_for_calculating_variance#Na%C3%AFve_algorithm

The naive one with f64 should be good enough. Or Welford with f32, but maybe costlier due to the division.